### PR TITLE
Fix enabled config to handle conditionals

### PR DESCRIPTION
**Pull request by nszoni**
**Are you a current Fivetran customer?** 

No, I'm Sonny Nguyen, Analytics Engineer from [Hiflylabs](https://hiflylabs.com):)

**What change(s) does this PR introduce?** 

Fixes a bug switching off the `apple_search_ads.search_term_report` source. Previously, it wasn't able to compile the Jinja expression because the `and` parameter was outside the context. This achieves the intended behaviour. 

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)

It only disables the source when the right `vars` are defined as False as intended.

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature fivetran/dbt_ad_reporting/issues/73
- [ ] No 

**How did you test the PR changes?** 

 

- [ ] Buildkite  
- [x] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [ ] Redshift
- [ ] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# dbt_apple_search_ads_source v0.2.1
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:1]: This looks great, would you also be able to add a [Contribors](https://github.com/fivetran/dbt_ad_reporting/blob/main/CHANGELOG.md#contributors) section to this release to give yourself contributor credit for your PR. You can see how this has been done within a few of our other package CHANGELOGS.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
This looks great @nszoni! Thanks so much for taking the time to contribute back to our packages to fix the issue you originally discovered! 🏅 

I am approving this PR, but I have one small request for you to add a contributor section to the CHANGELOG. Once that is applied I will be able to merge this and incorporate in the next patch release!

File: CHANGELOG.md (hunk 1 above)
Comment (nszoni) [reply CHANGELOG.md:1]: Just added the section!:)

**Comment by fivetran-joemarkiewicz** [comment]
Brilliant!! Thanks so much for the quick turnaround @nszoni! These updates look great to me and I will kick off our internal process to merge and cut the release 🎉

**Pull request merged**